packages/next/src/server/next-server.ts

```diff
@@ -306,11 +306,15 @@ export default class NextNodeServer extends BaseServer<
     // development.
   }
 
-  protected async loadInstrumentationModule() {
-    if (
+  private isProductionInstrumentationEnabled = () => {
```

Comment:
nit: looks like we're usually preferring methods over function properties in these classes